converters/fromZigbee.js

```diff
@@ -3428,6 +3435,43 @@ const converters = {
         },
     },
 
+    legrand_binary_input_moving: {
+        cluster: 'genBinaryInput',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options) => {
+            return {
+                moving: msg.data.presentValue,
```

Comment:
What does this represent?

Reply:
Nvm, I think is also an action?

Reply:
presentValue contains a boolean. Currently I'm supposing it's true when the shutter is moving, false when it's not but not 100% sure. Only time will confirm as after an OTA it had weird comportments.

Reply:
So if this boolean is true, the action should be moving, otherwise stopped.

Reply:
Changed in the last commit